wordpress-mobile/WordPress-iOS
# Move github helper functions to the shared repository

This PR removes the github-helper.rb script from this repository and updates Fastlane to use the plugin-version of the script that lives in the new shared repository _release-toolkit_. 

The new Fastfile defines three lanes that wrap the calls to the plugin.

To Test:
1. run `rake dependencies`;
2. copy `Script/fastlane/env.example` to `~/.wpios-env.default` (note that's a hidden file); 
3. set file permission to `600`; 
3. fill the `GHHELPER_ACCESS=` var in the `~/.wpios-env.default` with the GitHub access token (like `GHHELPER_ACCESS=lkjlkjfdsfhkj`: no brackets or marks required);
4. cd into the `Scripts` folder;
5. run `bundle exec fastlane set_frozen milestone:xxx`, with xxx = GitHub milestone number, and verify that the frozen tag is added to the milestone;
6. run `bundle exec fastlane unset_frozen milestone:xxx`, with xxx = GitHub milestone number, and verify that the frozen tag is removed from the milestone;
7. run `bundle exec fastlane protect_branch branch:xxxxx`, with xxxxx = branch name, and verify that the right protection is added to the branch on GitHub.
